src/PuliTemplateLoader.php

```diff
@@ -125,4 +125,28 @@ public function isFresh($path, $time)
             throw new Twig_Error_Loader($e->getMessage(), -1, null, $e);
         }
     }
+
+    /**
+     * Check if we have the source code of a template, given its name.
+     *
+     * @param string $name The name of the template to check if we can load
+     *
+     * @return bool If the template source code is handled by this loader or not
+     */
+    public function exists($name)
+    {
+        try {
+            $file = $this->repo->get($name);
```

Comment:
the issue is that if it happens, this will break things badly for twig users, forbidding to fallback to the next loader